.github/workflows/ruby-ci.yml

```diff
@@ -17,26 +17,20 @@ jobs:
     strategy:
       matrix:
         version:
-          - 2.6
-          - 2.7
-          - 3.0
           - 3.1
           - 3.2
+          - 3.3
         gemfile:
-          - gemfiles/rails-6-0.gemfile
+          - gemfiles/rails-7-0.gemfile
           - gemfiles/rails-latest-release.gemfile
           - gemfiles/rails-edge.gemfile
         exclude:
-          - version: 2.6
-            gemfile: gemfiles/rails-edge.gemfile
-          - version: 2.7
-            gemfile: gemfiles/rails-edge.gemfile
-          - version: 3.0
-            gemfile: gemfiles/rails-6-0.gemfile
           - version: 3.1
-            gemfile: gemfiles/rails-6-0.gemfile
+            gemfile: gemfiles/rails-edge.gemfile
           - version: 3.2
-            gemfile: gemfiles/rails-6-0.gemfile
+            gemfile: gemfiles/rails-7-0.gemfile
+          - version: 3.3
+            gemfile: gemfiles/rails-7-0.gemfile
```

Comment:
I don't have context on Rails and Ruby version compatibility. This [compatibility chart](https://www.fastruby.io/blog/ruby/rails/versions/compatibility-table.html) suggests that Ruby 3.2 and Ruby 3.3 are compatible with Rails 7.

I can [confirm](https://github.com/rails/rails/blob/main/rails.gemspec#L12) that Ruby 3.1 support was dropped on the latest rails main 

<img width="1371" alt="image" src="https://github.com/user-attachments/assets/e40fb662-dc2b-449b-832a-b1e1d2352475">

Reply:
Might be worth trying to remove the `exclude` block altogether. It was for incompatible Ruby + Rails combos but not many exist at this point other than Ruby 3.1 + Rails 8 (not yet out, so, main).

Mostly curious why we need Ruby 3.2 and 3.3 + Rails 7.